src/ProtoValidate/Internal/Evaluator/EvaluatorBuilder.cs

```diff
@@ -155,6 +158,9 @@ private void ProcessFields(MessageDescriptor messageDescriptor, MessageEvaluator
         foreach (var fieldDescriptor in fieldDescriptors)
         {
             var fieldConstraints = ConstraintResolver.ResolveFieldConstraints(fieldDescriptor);
+
+            if (fieldConstraints is null) continue;
```

Comment:
The short-circuiting with the null check here causes a number of conformance tests to fail.